pro/src/screens/OfferType/__tests-utils__/offerTypeUtils.ts

```diff
@@ -0,0 +1,11 @@
+import { screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+export const getEducationalOfferButton = (): HTMLInputElement => 
+  screen.getByLabelText("Une offre à destination d'un groupe scolaire") as HTMLInputElement
+
+export const getSubmitButton = (): HTMLButtonElement => 
+    screen.getByText('Étape suivante') as HTMLButtonElement
+
+export const clickOnEducationalType = (educationalOfferButton: HTMLInputElement): void => 
+  userEvent.click(educationalOfferButton)
```

Comment:
1. on peut regrouper les éléments commun au sein d'un seul utilitaire, c'est plus facile à utiliser, importer et faire évoluer
2. on essai toujours de sélectionner par role quand c'est possible (`getByRole`) c'est plus précis et ça permet aussi de s'assurer que la sémantique est bonne (qu'on a pas des `onClick` sur des divs par exemple)
3. je pense qu'on peut garder le userEvent dans le test directement

```suggestion
type GetElements : () => {
  submitButton: HTMLButtonElement,
  educationalOfferButton: HTMLInputElement,
}

export const getElements: GetElements = () => ({
  submitButton: screen.getByRole('button', { name: 'Étape suivante' }) as HTMLButtonElement,
  educationalOfferButton: screen.getByLabelText("Une offre à destination d'un groupe scolaire") as HTMLInputElement
})
```